<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageDialog</class>
 <widget class="QDialog" name="ImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="previousButton">
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="addTagButton">
        <property name="text">
         <string>Add Tag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateTagButton">
        <property name="text">
         <string>Edit Tag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeTagButton">
        <property name="text">
         <string>Remove Tag</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="addImageButton">
        <property name="text">
         <string>Add Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeImageButton">
        <property name="text">
         <string>Remove Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>50</y>
     <width>241</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QListWidget" name="tagList">
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Images: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imagesLabel">
        <property name="text">
         <string>#</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
